fastly/resource_fastly_service_v1.go

```diff
@@ -1319,7 +1332,7 @@ func resourceServiceV1Create(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*FastlyClient).conn
 	service, err := conn.CreateService(&gofastly.CreateServiceInput{
 		Name:    d.Get("name").(string),
-		Comment: "Managed by Terraform",
+		Comment: d.Get("name").(string),
```

Comment:
This looks like the wrong field value is being used and the same is present in another PR addressing comments.